collect_app/src/main/java/org/odk/collect/android/application/TestCollect.java

```diff
@@ -12,12 +10,6 @@
 
 public class TestCollect extends Collect {
 
-    @Override
-    protected RefWatcher setupLeakCanary() {
```

Comment:
Huh so LeakCanary won't be running in any test build automatically now?

Reply:
That's what I understand https://square.github.io/leakcanary/recipes/#running-leakcanary-in-instrumentation-tests

Reply:
So much better!